src/app/static/src/app/components/plots/bubble/SizeLegend.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+    <l-control position="bottomleft">
+        <div class="map-control p-1">
+            <svg :width="width" :height="height">
+                <circle v-for="(circle, index) in circles" :key="'circle-' + index" stroke="#aaa" stroke-width="1" fill-opacity="0"
+                        :r="circle.radius" :cx="circle.x" :cy="circle.y"></circle>
+                <text v-for="(circle, index) in circles" :key="'text-' + index" text-anchor="middle"
+                      :x="circle.textX" :y="circle.textY">{{circle.text}}</text>
+            </svg>
+        </div>
+    </l-control>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import {LControl} from "vue2-leaflet";
+    import {getRadius} from "./utils";
+    import {NumericRange} from "../../../types";
+
+    interface Circle {
+        x: number
+        y: number,
+        radius: number,
+        text: string,
+        textX: number,
+        textY: number
+    }
+
+    interface Props {
+        indicatorRange: NumericRange,
+        minRadius: number,
+        maxRadius: number
+    }
+
+    interface Computed {
+        circles: Circle[],
+        width: number,
+        height: number
+    }
+
+    const numeral = require('numeral');
+
+    export default Vue.extend<{}, {}, Computed, Props>({
+        name: "SizeLegend",
+        props: {
+            "indicatorRange": Object,
+            "minRadius": Number,
+            "maxRadius": Number
+        },
+        components: {
+            LControl
+        },
+        computed: {
+            width: function() {
+                return (this.maxRadius * 2) + 2; //leave room for stroke width
+            },
+            height: function() {
+                return (this.maxRadius * 2) + 10; //leave room for the top text
+            },
+            circles: function() {
+                const x = this.width / 2;
+                const circleFromRadius = (r: number, value: number, under: boolean = false) => {
+                    const y = this.height - r;
+
+                    let text = value > 1000 ? numeral(value).format("0a") : (+value.toFixed(2)).toString();
+                    if (under && text != "0"){
+                        text  = "<" + text;
+                    }
+
+                    return {x: x,  y: y, radius: r, text: text, textX: x, textY: y-r}
+                };
+
+                //We treat the minimum circle differently, since the smallest radius is actually likely to cover quite
+                //a wide range of low outliers, so we show the value for the next pixel up and prefix with '<'
+                const nextMinRadius = this.minRadius + 1;
+                const valueScalePoint = (Math.pow(nextMinRadius,2) - Math.pow(this.minRadius,2))/
```

Comment:
Can this line and the next be pulled out into their own util methods to make this function a bit easier to read?